server/routes/assessmentsRoute.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express')
+const assessmentsRouter = express.Router()
+const assessmentsDb = require('./assessmentsDb')
+
+router.get('/:id', (req, res) => {
```

Comment:
I have a sneaking suspicion this code won't work as `router` isn't defined anywhere.